### PR TITLE
Create de_de.json

### PR DESCRIPTION
**Pull request by offron**
Added German translation. If you have problem with umlauts ( Ä Ö Ü ) etc. let me know and i try to fix it

### CODE REVIEW

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "itemGroup.pipez": "Röhrez",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:2]: Mod names shouldn't be renamed

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  "block.pipez.item_pipe": "Item Röhre",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:3]: "Röhre" doesn't fit very well. I would suggest something like "Rohr" or "Leitung"

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  "block.pipez.fluid_pipe": "Flüssigkeits Röhre",
+  "block.pipez.energy_pipe": "Energie Röhre",
+  "block.pipez.universal_pipe": "Universele Röhre",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:6]: Universelle

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  "block.pipez.gas_pipe": "Gas Röhre",
+  "item.pipez.basic_upgrade": "Normale Röhre Verbesserung",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:8]: That's not proper german

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  "item.pipez.improved_upgrade": "Verbesserte Röhre Verbesserung",
+  "item.pipez.advanced_upgrade": "Erweiterte Röhre Verbesserung",
+  "item.pipez.ultimate_upgrade": "Ultimative Röhre Verbesserung",
+  "item.pipez.infinity_upgrade": "Unendliche Röhre Verbesserung",
+  "item.pipez.wrench": "Röhren Zange",
+  "item.pipez.filter_destination_tool": "Filter Ziel Gerät",
+  "tooltip.pipez.wrench": "Klicke auf eine Röhre um sie einzustellen",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:15]: "Klicke auf eine Röhre, um sie einzustellen"

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  "tooltip.pipez.distribution.furthest": "Weiteste zuert",
+  "tooltip.pipez.distribution.round_robin": "Rundlauf",
+  "tooltip.pipez.distribution.random": "Zufällig",
+  "tooltip.pipez.redstone_mode": "Rotstein Modus: %s",
+  "tooltip.pipez.redstone_mode.ignored": "Ignoriert",
+  "tooltip.pipez.redstone_mode.off_when_powered": "Aus Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.on_when_powered": "An Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.always_off": "Immer Aus",
+  "tooltip.pipez.filter_mode": "Filter Modus: %s",
+  "tooltip.pipez.filter_mode.whitelist": "Erlaubt",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:27]: This translation does not fit that well

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+  "tooltip.pipez.filter_mode.blacklist": "Nicht Erlaubt",
+  "tooltip.pipez.filter.not": "Nicht",
+  "tooltip.pipez.filter.accepts_tag": "Alle %s",
+  "tooltip.pipez.filter.item_tag.description": "Tags Präfix mit # oder item IDs",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:31]: Inconsistent capitalization
Also not a proper sentence

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+  "tooltip.pipez.filter.nbt_string.description": "NBT Data",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:32]: Not translated

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT tags haben",
+  "tooltip.pipez.filter.nbt.exact": "Genaue übereinstimmung NBT data",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:34]: Wrong capitalization

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+  "tooltip.pipez.filter.nbt.not_exact": "Nur bereitgestellte Tags abgleichen ",
+  "tooltip.pipez.filter.not_inverted": "Regulär",
+  "tooltip.pipez.filter.inverted": "Invertiert",
+  "tooltip.pipez.filter_destination_tool": "Klicke auf einen Block, um das Ziel festzulegen",
+  "tooltip.pipez.filter_destination_tool.destination": "Ziel: X=%s, Y=%s, Z=%s, %s",
+  "tooltip.pipez.filter.destination.description": "Benutze das Filter Ziel Gerät um ein Ziel festzulegen",
+  "tooltip.pipez.filter.destination.click_to_remove": "Klicken zum entfernen",
+  "tooltip.pipez.filter.unknown_block": "Unbekannter BLOCK",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:42]: Why "BLOCK"?

**Review by henkelmax — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 1 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:2]: Mistake from replacing words.

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 2 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:3]: will change to Rohr

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 3 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:6]: sorry for typo

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 4 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:8]: energie = Strom Rohr
Universelles = Rohr 
etc?

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 5 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:15]: How would you change it?

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 6 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:27]: whole Part?

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 7 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:31]: Didnt know the exact usage of it. Mind helping me out?

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 9 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:34]: will fix

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 10 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:42]: it was written all cap in Original. Will change

**Comment by offron** [comment]
Im sorry, this is my first time using Github and writing a translation. I will fix it right away. 
According to your name, i assume you are a native german speaker?

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 4 above)
Comment (henkelmax) [reply src/main/resources/assets/pipez/lang/de_de.json]: I'm talking about the upgrade names

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 6 above)
Comment (henkelmax) [reply src/main/resources/assets/pipez/lang/de_de.json:27]: I think leaving Blacklist and Whitelist would be best... I don't think there is a german word that would describe it good enough

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 7 above)
Comment (henkelmax) [reply src/main/resources/assets/pipez/lang/de_de.json]: "Item IDs oder Tags mit #"

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 4 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json]: changed to "modifikation" should fit better

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 11]
```diff
@@ -5,11 +5,11 @@
   "block.pipez.energy_pipe": "Strom Rohr",
   "block.pipez.universal_pipe": "Universelle Rohr",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:6]: "Universelles Rohr"

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 12]
```diff
@@ -1,45 +1,45 @@
...
+  "block.pipez.fluid_pipe": "Flüssigkeits Rohr",
+  "block.pipez.energy_pipe": "Strom Rohr",
+  "block.pipez.universal_pipe": "Universelle Rohr",
+  "block.pipez.gas_pipe": "Gas Rohr",
+  "item.pipez.basic_upgrade": "Normale Rohr Verbesserung",
+  "item.pipez.improved_upgrade": "Verbesserte Rohr Verbesserung",
+  "item.pipez.advanced_upgrade": "Fortgeschrittene Rohr Verbesserung",
+  "item.pipez.ultimate_upgrade": "Ultimative Rohr Verbesserung",
+  "item.pipez.infinity_upgrade": "Unendliche Rohr Verbesserung",
   "item.pipez.wrench": "Röhren Zange",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:13]: "Schraubenschlüssel"

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 13]
```diff
@@ -1,45 +1,45 @@
...
   "tooltip.pipez.filter_mode.blacklist": "Nicht Erlaubt",
   "tooltip.pipez.filter.not": "Nicht",
   "tooltip.pipez.filter.accepts_tag": "Alle %s",
-  "tooltip.pipez.filter.item_tag.description": "Tags Präfix mit # oder item IDs",
-  "tooltip.pipez.filter.nbt_string.description": "NBT Data",
-  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT tags haben",
-  "tooltip.pipez.filter.nbt.exact": "Genaue übereinstimmung NBT data",
+  "tooltip.pipez.filter.item_tag.description": "Präfix Stichworte mit # oder Item IDs",
+  "tooltip.pipez.filter.nbt_string.description": "NBT Daten",
+  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT Stichworte haben",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:33]: "kann"

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 14]
```diff
@@ -1,45 +1,45 @@
...
   "tooltip.pipez.distribution": "Verteilung: %s",
   "tooltip.pipez.distribution.nearest": "Nächste zuerst",
   "tooltip.pipez.distribution.furthest": "Weiteste zuert",
   "tooltip.pipez.distribution.round_robin": "Rundlauf",
   "tooltip.pipez.distribution.random": "Zufällig",
   "tooltip.pipez.redstone_mode": "Rotstein Modus: %s",
   "tooltip.pipez.redstone_mode.ignored": "Ignoriert",
-  "tooltip.pipez.redstone_mode.off_when_powered": "Aus Wenn Aktiviert",
-  "tooltip.pipez.redstone_mode.on_when_powered": "An Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.off_when_powered": "Aus wenn Redstone aktiviert",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:23]: Inconsistent capitalization (Aktiviert/aktiviert)

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 15]
```diff
@@ -1,45 +1,45 @@
...
   "tooltip.pipez.filter.not": "Nicht",
   "tooltip.pipez.filter.accepts_tag": "Alle %s",
-  "tooltip.pipez.filter.item_tag.description": "Tags Präfix mit # oder item IDs",
-  "tooltip.pipez.filter.nbt_string.description": "NBT Data",
-  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT tags haben",
-  "tooltip.pipez.filter.nbt.exact": "Genaue übereinstimmung NBT data",
+  "tooltip.pipez.filter.item_tag.description": "Präfix Stichworte mit # oder Item IDs",
+  "tooltip.pipez.filter.nbt_string.description": "NBT Daten",
+  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT Stichworte haben",
+  "tooltip.pipez.filter.nbt.exact": "Genaue Übereinstimmung der Stichwort Daten",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:34]: "Stichwort"??

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 16]
```diff
@@ -1,45 +1,45 @@
...
   "tooltip.pipez.filter.not_inverted": "Regulär",
   "tooltip.pipez.filter.inverted": "Invertiert",
   "tooltip.pipez.filter_destination_tool": "Klicke auf einen Block, um das Ziel festzulegen",
   "tooltip.pipez.filter_destination_tool.destination": "Ziel: X=%s, Y=%s, Z=%s, %s",
   "tooltip.pipez.filter.destination.description": "Benutze das Filter Ziel Gerät um ein Ziel festzulegen",
   "tooltip.pipez.filter.destination.click_to_remove": "Klicken zum entfernen",
-  "tooltip.pipez.filter.unknown_block": "Unbekannter BLOCK",
+  "tooltip.pipez.filter.unknown_block": "UNBEKANTER BLOCK",
   "tooltip.pipez.filter.destination_location": "Bei X=%s, Y=%s, Z=%s",
   "tooltip.pipez.filter.destination_distance": "%s blöcke(s) entfernt",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:44]: "blöckes"??
Not captialized

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 17]
```diff
@@ -1,45 +1,45 @@
...
   "tooltip.pipez.filter.destination_side": "Zugewandt %s",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:45]: Doesn't really fit

**Review by henkelmax — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 15 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:34]: was the straight translation. Should i leave "tag"?

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 16 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:44]: I used the en_us.json as template, there it was: `` "tooltip.pipez.filter.unknown_block": "UNKNOWN BLOCK",``
I changed it now to ``"tooltip.pipez.filter.unknown_block": "Unbekannter Block"``

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 17 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:45]: Which part do you mean? i fixed the typo aswell.

First of all, im realy sorry for all the misstakes. I realy need to learn to pay more attention and not rush it. 
And i realy realy apprechiate your patience with me.
Im translating with notepad++ would there be an better alternative?

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 15 above)
Comment (henkelmax) [reply src/main/resources/assets/pipez/lang/de_de.json]: Yeah

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 17 above)
Comment (henkelmax) [reply src/main/resources/assets/pipez/lang/de_de.json:45]: "Zugewandt"

No worries :D
VSCode is a good alternative.

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 15 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json]: done

File: src/main/resources/assets/pipez/lang/de_de.json (hunk 17 above)
Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:45]: would "Zielseite" work?

Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:45]: what does it "destination_side" stand for? where is it used?

Comment (henkelmax) [reply src/main/resources/assets/pipez/lang/de_de.json:45]: It is used for the side of the attached block with the destination tool.

Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json:45]: what would you recomend? "richtung" ?

Comment (henkelmax) [reply src/main/resources/assets/pipez/lang/de_de.json:45]: Yeah, "Richtung" sounds good. The only alternative I had in mind would be "Orientierung"

Comment (offron) [reply src/main/resources/assets/pipez/lang/de_de.json]: I like "Orientierung" more, i changed it to that. I pushed now the latest changes.

File: src/main/resources/assets/pipez/lang/de_de.json [hunk 18]
```diff
@@ -0,0 +1,85 @@
...
+  "tooltip.pipez.filter_mode": "Filter Modus: %s",
+  "tooltip.pipez.filter_mode.whitelist": "Whitelist",
+  "tooltip.pipez.filter_mode.blacklist": "Blacklist",
+  "tooltip.pipez.filter.not": "Nicht",
+  "tooltip.pipez.filter.accepts_tag": "Alle %s",
+  "tooltip.pipez.filter.item_tag.description": "Item IDs oder Tags mit #",
+  "tooltip.pipez.filter.nbt_string.description": "NBT Daten",
+  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kann keine NBT Tags haben",
+  "tooltip.pipez.filter.nbt.exact": "Genaue Übereinstimmung der Tags",
+  "tooltip.pipez.filter.nbt.not_exact": "Nur bereitgestellte Tags abgleichen ",
```
Comment (henkelmax) [inline src/main/resources/assets/pipez/lang/de_de.json:35]: Unnecessary space at the and

**Review by henkelmax — CHANGES_REQUESTED** [review]

**Comment by henkelmax** [comment]
Thank you very much for the translation!

**Pull request merged**